data/unknown watchers.txt

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2016 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Once the player has taken quarg technology, they will be watched wherever they go.
+government "Unknown Watchers"
+	swizzle 0
+	color .88 .77 0
+	"player reputation" 1
+	"friendly hail" "encrypted signal"
+	"hostile hail" "encrypted signal"
+
+phrase "encrypted signal"
+	word
+		"[0100010110101101011010100101]"
+		"[white noise]"
+		"[00000010000001100000111000011110001001]"
+		"[static]"
+
+phrase "unknown watcher names"
+	word
+		"Mysterious"
+		"Unusual"
+		"Strange"
+		"Unidentified"
+	word
+		" Drone"
+		" Observer"
+		" Flying Object"
+
+ship "Drone (Anonymous)"
+	sprite "ship/surveillance drone"
+	attributes
+		category "Interceptor"
```

Comment:
It is an automation(line 53).  It unfortunately cannot have the "Fighter" or "Drone" category and still escort a ship that has no fighter/drone bay for it to occupy (this is a hard-coded restriction that I didn't feel like working around). And I didn't want to invent a new ship category just for these things (the list of acceptable ship categories is also hard coded).